borg/helpers.py

```diff
@@ -269,15 +269,32 @@ def list_archive_infos(self, sort_by=None, reverse=False):
         return archives
 
 
-def prune_within(archives, within):
+def within_range(s):
```

Comment:
the "within" somehow comes from the main user of this code "prune within", but is not related to this validator.

so rather name it "interval" or so?